Add Blzmod example to pull.md

jjmaestro opened the pull request:
The documentation only had `WORKSPACE.bazel` examples, so I'm adding one for `MODULE.bazel`.

The example I'm using also has a slight "issue" with the architecture variant that (I think) can easily to trip people (like it tripped me).

jjmaestro commented:
@thesayyn / @alexeagle this should be a quick review / stamp, it's a simple doc change that I think can be useful for n00bs like me that are new to Bazel and have been using Blzmod in a clean repo 😄 

Thanks!!

thesayyn commented:
I believe you should update the doc in oci/pull.bzl instead of directly editing the generated file.

jjmaestro commented:
@thesayyn eeeh... you are right, of course! dunno how I missed that 😅  I've added the changes to the .bzl file and re-rendered the docs (and rebased the branch). I think it should be ready now 😄  Thanks!

thesayyn commented on oci/pull.bzl:
````diff
@@ -23,17 +23,57 @@ oci_pull(
+)
+```
+
+Typical usage in `MODULES.bazel`:
````
MODULE.bzl
